src/verilog.y

```diff
@@ -40,6 +40,11 @@
 #define BBUNSUP(fl, msg) (fl)->v3warn(E_UNSUPPORTED, msg)
 #define GATEUNSUP(fl, tok) \
     { BBUNSUP((fl), "Unsupported: Verilog 1995 gate primitive: " << (tok)); }
+#define PRIMDLYUNSUP(nodep) \
+    if (nodep) { \
+        nodep->v3warn(ASSIGNDLY, "Unsupported: Ignoring delay on this primitive."); \
```

Comment:
Need test for this please.

Reply:
Yes, and please put {} around the if statement to prevent bugs if the macro is misused. Also some lint tools check for those for that reason.